recipes/hjson/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "hjson" %}
+{% set version = "3.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hjson-{{ version }}.tar.gz
+  sha256: 2838fd7200e5839ea4516ece953f3a19892c41089f0d933ba3f68e596aacfcd5
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
```

Comment:
Upstream uses the "scripts" tag so this cannot be noarch: python. I would recommend filing an issue upstream, asking them to change it to console_scripts entry_points: https://github.com/hjson/hjson-py/blob/20a809beedfc05255c3d5c79dee6f4d1b7ea46f1/setup.py#L75

Reply:
Hmm, I might just make it a non-noarch package for now. The hjson-py repo hasn't had a commit in 1.5 years so it might take a while to get this done on upstream quickly.

Reply:
You can also make a PR and add that as a patch until that is accepted but yes arch is probably the fastest and easiest way.

Reply:
I see there are related issues at https://github.com/hjson/hjson-py/issues/20 and https://github.com/hjson/hjson-py/issues/22. Will try and submit a patch.